src/init/features/firestore/indexes.ts

```diff
@@ -53,6 +53,9 @@ async function getIndexesFromConsole(
     const res = await Promise.all([indexesPromise, fieldOverridesPromise]);
     return JSON.stringify(indexes.makeIndexSpec(res[0], res[1]), null, 2);
   } catch (e: any) {
+    if (e.status === 404 || e.code === 403) {
```

Comment:
Catching 403s seems possibly unsafe - won't this catch and silence cases where the user does not have the required permission?